features/importing_service_definitions.feature

```diff
@@ -10,7 +10,7 @@ Feature: Importing service definitions
       extensions:
         Zalas\Behat\NoExtension:
           imports:
-            - features/bootstrap/config/services.yml
+            - %paths.base%/features/bootstrap/config/services.yml
```

Comment:
I guess I'm fine with adding the fix, but I'd keep this example as it was. I'd rather encourage simpler config (`%paths.base%` is technically not needed as everything is resolved relative to this path).

Alternatively, instead of supporting this single parameter (what if someone tries to use other Behat parameters?), we could detect the situation and throw a better exception saying that parameters are not expanded in import paths.